src/appleseed-max-impl/appleseedrenderer/projectbuilder.h

```diff
@@ -54,6 +55,7 @@ foundation::auto_release_ptr<renderer::Project> build_project(
     const std::vector<DefaultLight>&    default_lights,
     const ViewParams&                   view_params,
     const RendParams&                   rend_params,
+    INode*                              view_node,
```

Comment:
Move `view_node` before `view_params` to be consistent with the argument order you chose for `build_camera()` (which makes sense in my opinion).